bokeh/models/renderers.py

```diff
@@ -17,6 +17,7 @@ class Renderer(PlotObject):
     generally useful to instantiate on its own.
 
     """
+    hide = Bool(False)
```

Comment:
I though we will be only hiding the axis... but this would potentially enable the hiding in any renderer... then, is this PR pretending a broader scope for hiding? ping @bryevdv for his thoughts...